helm/charts/determined/templates/master-permissions.yaml

```diff
@@ -10,35 +10,58 @@ metadata:
 
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind: Role
 metadata:
   name: determined-master-{{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
   labels:
      app: determined-master-{{ .Release.Name }}
      release: {{ .Release.Name }}
 rules:
-  - apiGroups: [""]
-    resources: ["pods", "pods/status", "pods/log", "configmaps"]
```

Comment:
We have added `pods/status` now to the cloud user RBAC. We should try to leave these in and see if it works.